<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q::vtk::InteractorStyleNavigation</class>
 <widget class="QWidget" name="Q::vtk::InteractorStyleNavigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InteractorStyleNavigation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxX"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxY"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxZ"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
